p2p/pkg/preconfirmation/tracker/tracker.go

```diff
@@ -87,7 +87,7 @@ func NewTracker(
 		preconfContract: preconfContract,
 		receiptGetter:   receiptGetter,
 		optsGetter:      optsGetter,
-		newL1Blocks:     make(chan *blocktracker.BlocktrackerNewL1Block),
+		newL1Blocks:     make(chan *blocktracker.BlocktrackerNewL1Block, 5),
```

Comment:
Should we declare those parameters as constants and add a comment explaining why 5 and not 10, for example?